books/free-programming-books-subjects.md

```diff
@@ -837,6 +837,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [How to deal with Passwords](https://github.com/MHM5000/pass)
 * [Information Security Management](https://www.infosecuritybook.com) - Marcos Sêmola (PDF)
 * [Intrusion Detection Systems with Snort](https://ptgmedia.pearsoncmg.com/images/0131407333/downloads/0131407333.pdf) (PDF)
+* [Linux Basics for Hackers](https://www.kea.nu/files/textbooks/humblesec/linuxbasicsforhackers.pdf) - OccupyTheWeb (PDF)
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.